src/mesh/curved_mesh.jl

```diff
@@ -42,6 +42,10 @@ Create a CurvedMesh of the given size and shape that uses `RealT` as coordinate
 function CurvedMesh(cells_per_dimension, faces; RealT=Float64, unsaved_changes=true, faces_as_string=faces2string(faces))
   NDIMS = length(cells_per_dimension)
 
+  # After a mesh is loaded from a file, the functions defining it are evaluated in the function `load_mesh` using `eval`.
+  # If this function is used before the next top-level evaluation, this causes a world age problem.
+  Base.invokelatest(validate_faces, faces)
```

Comment:
This is a quick fix for the problems from the latest CI run. It works now, but it's just a temporary fix until we implement the global mapping.

Reply:
:+1: